app/src/pages/inside/common/launchSuiteGrid/hamburger/hamburger.jsx

```diff
@@ -167,15 +172,26 @@ export class Hamburger extends Component {
             )}
             <HamburgerMenuItem
               text={intl.formatMessage(messages.forceFinish)}
+              title={
+                (!canForceFinishLaunch(
```

Comment:
Please make it a class method and rewrite (maybe using `if..else` or temporary constants or both), it's unreadable now.

Reply:
Done